django/pyrog/migrations/0002_drop_template.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.1.2 on 2021-02-08 13:21
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("pyrog", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name="source",
+            unique_together=set(),
```

Comment:
Should the source name be unique?

Reply:
Possibly. If we make it unique, we'll need to right a data migration to respect the constraint (for the case where 2 sources from different templates have the same name).